app.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+var allLocations = [];
+var salesDataTable = document.getElementById('locationjs');
+//First and Pike
+function Location(minCustPerHour, maxCustPerHour, avgCookiesPerCust, locationName) {
+
+  this.minCustPerHour = minCustPerHour;
+  this.maxCustPerHour = maxCustPerHour;
+  this.avgCookiesPerCust = avgCookiesPerCust;
+  this.randCustomersPerHour = [];
+  this.totalCookiesSoldPerHour = [];
+  this.totalDailyCookiesSold = 0;
+  this.locationName = locationName;
+  this.calcRandCustomersPerHour = function() {
+    for (var i = 0; i < hours.length; i++) {
+      this.randCustomersPerHour.push(Math.ceil(Math.random() * (this.maxCustPerHour - this.minCustPerHour + 1)) + this.minCustPerHour);
+
+    }
+  };
+
+
+  this.calctotalCookiesSoldPerHour = function() {
+    this.calcRandCustomersPerHour();
+    for (var i = 0; i < hours.length; i++) {
+      this.totalCookiesSoldPerHour.push(Math.ceil(this.randCustomersPerHour[i] * this.avgCookiesPerCust));
+      this.totalDailyCookiesSold += this.totalCookiesSoldPerHour[i];
```

Comment:
For your bug, I think somehow this is calculating every time you hit submit, so it keeps adding it over and over again to your this.totalDailyCookiesSold in each of your instances of stores.